recipes/sagemaker-studio-image-build/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "sagemaker-studio-image-build" %}
+{% set version = "0.6.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sagemaker_studio_image_build-{{ version }}.tar.gz
+  sha256: b7586597ca2395a63e463f8e3ab981ae987d5ea500bf86096af9a79dc41ec9b5
+
+build:
+  number: 0
+  entry_points:
+    - sm-docker=sagemaker_studio_image_build.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [not win]
```

Comment:
Right now this says "Skip everything EXCEPT Windows", so I don't think that's what you want?

Reply:
whoops. What's the syntax for the opposite: skip windows ?
copying https://github.com/conda-forge/sagemaker-python-sdk-feedstock/blob/master/recipe/meta.yaml#L17
does the text after the # do anything? For exaple sagemaker-python-sdk has # [win]

Reply:
Yes, after the `#` is the selector, that says what conditions `skip: true` applies to. So if you want to skip Windows, then it should be `skip: true  # [win]`

Reply:
Cheers. I naively assumed it was a comment